docs-src/go/how-to-list-workflow-executions-using-the-client-in-go.md

```diff
@@ -9,4 +9,31 @@ tags:
   - client
 ---
 
-Use [`Client.ListWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow).
+The `ListWorkflow()` function retrieves [Workflow Executions](/concepts/what-is-a-workflow-execution) that match the properties specified in a [List Filter](/concepts/what-is-a-list-filter).
+These properties are defined by [Search Attributes](/concepts/what-is-a-search-attribute), which are used to retrieve Workflow Executions from the [Visibility store](/concepts/what-is-visibility).
+
+Use a List Filter to define a `request` to pass into `ListWorkflow()`.
+
+```go
+// ...
+request := "CloseTime = missing"
+```
+
+This `request` value will only return open Workflows.
+To list all Workflow Executions that satisfy this List Filter, use [`ListWorkflow()`](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow) on the [Temporal Client](/concepts/what-is-a-temporal-client).
```

Comment:
```suggestion
To list all Workflow Executions that satisfy this List Filter, use [ListWorkflow()](https://pkg.go.dev/go.temporal.io/sdk/client#Client.ListWorkflow) on the [Temporal Client](/concepts/what-is-a-temporal-client).
```

Reply:
This is a bit verbose and repetitive -
You could wipe this whole sentence if the API definition is linked in the first mention of the ListWorkflow API in the first sentence.